tests/unittests/CoreGraphics.Drawing/CGPathDrawingTests.cpp

```diff
@@ -579,7 +579,69 @@ DRAW_TEST_F(CGPath, FillArcsSimple, UIKitMimicTest<>) {
     CGPathRelease(thepath);
 }
 
-DRAW_TEST_F(CGPath, FillArcsComplex, UIKitMimicTest<>) {
+DRAW_TEST_F(CGPath, BoundingBoxes, UIKitMimicTest<>) {
```

Comment:
It is cool that this is a drawing test, but it can be validated completely mathematically. Regressions will be easier to catch then a drawing failure, because we won't be testing the entire graphics stack.